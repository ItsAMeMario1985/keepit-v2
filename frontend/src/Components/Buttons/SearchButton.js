import PropTypes from 'prop-types'
import styled from 'styled-components/macro'
import { useHistory, useLocation } from 'react-router-dom'
import { SearchIcon } from '../Icons'

export default function SearchButton({ onClick }) {
  return <SearchIcon onClick={onClick}></SearchIcon>
}
//
