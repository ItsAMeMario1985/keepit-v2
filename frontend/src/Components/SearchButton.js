import PropTypes from 'prop-types'
import styled from 'styled-components/macro'
import { ReactComponent as SearchSvg } from '../Assets/search.svg'
import { useHistory, useLocation } from 'react-router-dom'

export default function SearchButton({ onClick, buttonText }) {
  SearchButton.propTypes = {
    onClick: PropTypes.func.isRequired,
    buttonText: PropTypes.string.isRequired,
  }
  const history = useHistory()

  return <SearchSvg onClick={() => history.goBack()}></SearchSvg>
}

/*
import ImageUploading from 'react-images-uploading'
import { useHistory, useLocation } from 'react-router-dom'
import { ReactComponent as ActionButtonSvg } from '../Assets/keepit-button.svg'
import styled from 'styled-components/macro'

export default function UploadButtonFooter() {
  const history = useHistory()

*/
