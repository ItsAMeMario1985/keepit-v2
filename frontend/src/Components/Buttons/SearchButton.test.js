import { render } from '@testing-library/react'
import SearchButton from './SearchButton'
describe('SearchButton', () => {
  it('renders correctly', () => {
    const { container } = render(<SearchButton />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
