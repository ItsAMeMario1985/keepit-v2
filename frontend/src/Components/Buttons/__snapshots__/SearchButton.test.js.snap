// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchButton renders correctly 1`] = `
<svg>
  search.svg
</svg>
`;
